### PR TITLE
system/FreeFileSync: Updated for version 13.4.

### PR DESCRIPTION
**Pull request by antonioleal**
system/FreeFileSync: Updated for version 13.4.

### CODE REVIEW

**Comment by aclemons** [comment]
@sbo-bot: build system/FreeFileSync

**Comment by willysr** [comment]
@sbo-bot: build i586 system/FreeFileSync

**Comment by antonioleal** [comment]
The CI/CD is checking the md5 of a "download.php" file not the actual "FreeFileSync_13.4_Linux.tar.gz" source.

My automation is working fine, could you please check on your side?

File: system/FreeFileSync/FreeFileSync.SlackBuild [hunk 1]
```diff
@@ -25,7 +25,7 @@
 cd $(dirname $0) ; CWD=$(pwd)
```
Comment (aclemons) [inline system/FreeFileSync/FreeFileSync.SlackBuild:25]: These perms should not change from 0644

**Comment by aclemons** [comment]
```
# wget https://freefilesync.org/download/FreeFileSync_13.4_Linux.tar.gz
--2024-02-24 13:43:55--  https://freefilesync.org/download/FreeFileSync_13.4_Linux.tar.gz
Resolving freefilesync.org (freefilesync.org)... 104.21.2.160, 172.67.129.95, 2606:4700:3031::ac43:815f, ...
Connecting to freefilesync.org (freefilesync.org)|104.21.2.160|:443... connected.
HTTP request sent, awaiting response... 307 Temporary Redirect
Location: https://freefilesync.org/download.php [following]
--2024-02-24 13:43:56--  https://freefilesync.org/download.php
Reusing existing connection to freefilesync.org:443.
HTTP request sent, awaiting response... 200 OK
Length: unspecified [text/html]
Saving to: ‘FreeFileSync_13.4_Linux.tar.gz’

FreeFileSync_13.4_Linux.tar.gz                           [ <=>                                                                                                                   ]  52.76K  --.-KB/s    in 0.09s

2024-02-24 13:43:56 (574 KB/s) - ‘FreeFileSync_13.4_Linux.tar.gz’ saved [54026]
```

```
ls -la FreeFileSync_13.4_Linux.tar.gz
-rw-r--r-- 1 root root 54026 Feb 24 13:43 FreeFileSync_13.4_Linux.tar.gz
```

```
# file FreeFileSync_13.4_Linux.tar.gz
FreeFileSync_13.4_Linux.tar.gz: HTML document, Unicode text, UTF-8 text, with very long lines (303), with CRLF, LF line terminators
```

Downloading with wget results in an HTML file. Possibly their server is checking based on user agents. Is the tarball mirrored anywhere else which is more reliable?

**Comment by aclemons** [comment]
I tried it a few times more. I did get a 30M tarball a few times, so it seem that the server is unreliable.

**Comment by antonioleal** [comment]
Hi,
as fas as I know there is no other host for the binariy tarball.
Since is binary, better to use the original upstream anyaway.

wget https://freefilesync.org/download/FreeFileSync_13.4_Linux.tar.gz works fine.
 
should get you 30.1 MiB (31,603,402 bytes)

But I never got an html file.

File: system/FreeFileSync/FreeFileSync.SlackBuild (hunk 1 above)
Comment (antonioleal) [reply system/FreeFileSync/FreeFileSync.SlackBuild:25]: ok I done this change.

**Comment by antonioleal** [comment]
Ok I could reproduce it once now.
The upstream server did indeed result in a "html" file, I suppose they are having some problem now.

**Comment by aclemons** [comment]
@sbo-bot: build system/FreeFileSync

**Comment by aclemons** [comment]
@sbo-bot: build i586 system/FreeFileSync

**Comment by willysr** [comment]
i'll take it as closed for now

**Pull request closed without merging**